example/example.php

```diff
@@ -1,17 +1,19 @@
 <?php
 
+declare(strict_types=1);
+
 require_once realpath(__DIR__ . '/../vendor/autoload.php');
 
-/*
+/**
```

Comment:
```suggestion
/*
```

А зачем это изменение? Это же не DocBlock, а простой мультистрочный комментарий.